test/v1/controllers/CreateAndAmendEmploymentExpensesControllerSpec.scala

```diff
@@ -31,14 +31,19 @@ import play.api.libs.json.{JsValue, Json}
 import play.api.mvc.Result
 import v1.mocks.requestParsers.MockCreateAndAmendEmploymentExpensesRequestParser
 import v1.mocks.services.MockCreateAndAmendEmploymentExpensesService
-import v1.models.request.createAndAmendEmploymentExpenses.{CreateAndAmendEmploymentExpensesBody, CreateAndAmendEmploymentExpensesRawData, CreateAndAmendEmploymentExpensesRequest, Expenses}
+import v1.models.request.createAndAmendEmploymentExpenses.{
```

Comment:
This import can be simplified to: import v1.models.request.createAndAmendEmploymentExpenses._